fractal_tasks_core/lib_metadata_parsing.py

```diff
@@ -18,8 +18,9 @@
 import numpy as np
 import pandas as pd
 from defusedxml import ElementTree
+import matplotlib.pyplot as plt
```

Comment:
Not a critical issue, but if we want to have some graphical inspection tools I would rather keep them in an separated module.

Reply:
Separate module as in a separate lib python file, i.e. `lib_metadata_checks.py`? Or would you want to avoid this dependency for the task module?

Reply:
Separate lib python file.
I'd rather not have `import matplotlib` at the beginning of a core library module. If it is in a different file, we can one day just drop it if there is any problem.

Reply:
More on this:
we currently depend on matplotlib (indirectly), but we may hopefully get rid of it in the future. In that case, we may want to move it to a package extra.

Reply:
Sounds good, will update it that way :)